test/suites/basic.sh

```diff
@@ -126,7 +126,11 @@ test_basic_usage() {
 
   # Test container copy
   lxc copy bar foo
+  lxc config device set foo eth0 ipv4.address=192.0.2.2
+  # Test container with static IP copy
+  lxc copy foo baz
   lxc delete foo
+  lxc delete baz
 
```

Comment:
Can you add a test for restoring a snapshot that has a static IP assigned to a NIC that conflicts with another instance.
This checks for the first scenario described in https://github.com/lxc/lxd/issues/10114#issuecomment-1077449896

Reply:
Done